person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :age, :name
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = null
```

Comment:
- [ ] Please edit the id to be picked as a random number instead of being `null` for each object.
```@id = Random.rand(1..1000)```